composer/workflows/dataflowtemplateoperator_tutorial_test.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from airflow import models
+from . import unit_testing
```

Comment:
Sorry, where is the `unit_testing`? Also, ditto for the rules around import statements.

Reply:
`unit_testing` is part of the Composer `workflows` folder. It is what we use for unit testing DAG samples - it follows Airflow best practices which suggest testing a DAG's validity